assets/tests/components/notificationDrawer.test.tsx

```diff
@@ -30,13 +30,14 @@ describe("NotificationDrawer", () => {
     expect(dispatch).toHaveBeenCalledWith(closeNotificationDrawer())
   })
 
-  test("renders notifications", () => {
+  test("renders latest notification", () => {
```

Comment:
The notification drawer should show all the notifications, so I don't think this name should change.